examples/ensemble/plot_forest_importances_faces.py

```diff
@@ -3,46 +3,91 @@
 Pixel importances with a parallel forest of trees
 =================================================
 
-This example shows the use of forests of trees to evaluate the impurity-based
-importance of the pixels in an image classification task (faces).
-The hotter the pixel, the more important.
+This example shows the use of a forest of trees to evaluate
+the impurity based importance of the pixels in an image
+classification task on the faces dataset.
+The hotter the pixel, the more important it is.
 
 The code below also illustrates how the construction and the computation
 of the predictions can be parallelized within multiple jobs.
 """
+# %%
 print(__doc__)
 
-from time import time
 import matplotlib.pyplot as plt
 
+# %%
+# Loading the data and model fitting
+# ----------------------------------
+# We use the faces data from datasets submodules when using impurity-based
+# feature importance. One drawback of this method is that it cannot be
+# applied on a separate test set.
+# on a separate test set but for this example, we are interested
+# in representing the information learned from the full dataset.
+# Also, we'll set the number of cores to use for the tasks.
```

Comment:
I think talking about a test set here while the example does not split on a test set could be confusing to a reader. Here is my suggestion:

> First, we load the olivetti faces dataset and limit the dataset to contain only the first
> five classes. Then we train a random forest on the dataset and evaluate the
> impurity-based feature importance.

Reply:
@thomasjpfan Thanks a lot for the review! 
I think your explanation is much clearer than the previous one but I still think it is important to mention the drawback of using this method. Let me know what you think of the new version.